### ISSUE TITLE
Feature: list-trackers

### ISSUE THREAD

**Issue by maxtaco** [keybase/go#161]
Need:
  - [ ] standalone mode
  - [ ] RPC protocol
  - [ ] Improved server API endpoint; the current one is mainly for the Web site.
  - [ ] daemon-mode

**Comment by patrickxb** [comment]
Is this the equivalent of the node client list-tracking command?

**Comment by maxtaco** [comment]
No, that's #70.  This is list-trackERs, which the node client never bothered to implement

**Comment by patrickxb** [comment]
ok, got it.

**Comment by maxtaco** [comment]
(we'll have to work on the API stuff together, not sure what it should be...)
